grid.cpp

```diff
@@ -329,8 +331,13 @@ void initializeGrids(
       phiprof::stop("Init moments");
       */
 
+   } else if (P::writeFullBGB) {
```

Comment:
Suggestion for new comment:

If, instead of starting a regular simulation, we are only writing out the background field, it is enough to set a dummy load balance value of 1 here.